concent_api/core/views.py

```diff
@@ -380,16 +484,18 @@ def handle_receive_force_get_task_result_upload_for_requestor(decoded_message: m
     assert file_transfer_token.timestamp <= file_transfer_token.token_expiration_deadline  # pylint: disable=no-member
 
     force_get_task_result_upload = message.concents.ForceGetTaskResultUpload(
-        file_transfer_token   = file_transfer_token,
+        file_transfer_token = file_transfer_token,
     )
     force_get_task_result_upload.force_get_task_result = decoded_message.force_get_task_result
 
     store_message_and_message_status(
-        force_get_task_result_upload.TYPE,
-        force_get_task_result_upload.force_get_task_result.report_computed_task.task_to_compute.compute_task_def['task_id'],
-        force_get_task_result_upload.serialize(),
-        status    = ReceiveStatus,
-        delivered = True,
+        decoded_message.TYPE,
+        decoded_message.force_get_task_result.report_computed_task.task_to_compute.compute_task_def['task_id'],
+        decoded_message.serialize(),
```

Comment:
This looks like a bug. `decoded_message` already comes from the database and you're storing it again. Why? The code before this change was using `force_get_task_result_upload` here.

Reply:
I can't remember now, probably it was related with some failing test, but it works now on the old code so it looks like not necessary, I will remove it.